### ISSUE TITLE
Scale is not equivariantly handled

### ISSUE THREAD

**Issue by hongsukchoi** [dahiyaaneesh/peclr#3]
Hi, 

Thank you for your great work!

The paper says the proposed approach is equivariant to geometric transformations.
However, checking your code [here](https://github.com/dahiyaaneesh/peclr/blob/e835cbaa8cca25d74eb903b7276994c7424c779e/src/models/unsupervised/hybrid2_model.py#L27), only rotation and translation are consideration.

Where is the scale handled?

**Comment by dahiyaaneesh** [comment]
Hi @hongsukchoi 

Thanks for your interest in the paper. [Normalization](https://github.com/dahiyaaneesh/peclr/blob/master/src/models/unsupervised/hybrid2_model.py#:~:text=projection1%20%3D%20F.normalize,normalize(projections%5Bbatch_size%3A%5D)) of embeddings handles the scale.
